colibri/src/logging.rs

```diff
@@ -0,0 +1,44 @@
+use crate::args::Args;
+
+use tracing_appender::rolling::{RollingFileAppender, Rotation};
+use tracing_subscriber::filter::EnvFilter;
+
+// Config logging for the app. We allow to log to a system file
+// or to the stdout. If logs are stored in files they are rotated
+// daily and there is a max of `max_logfiles_num` files saved.
+pub fn config_logging(args: Args) {
+    let filter = EnvFilter::builder()
+        .with_default_directive(args.log_level.into())
+        .parse("")
+        .unwrap();
+
+    if !args.log_to_stdout {
+        let filename: String = args
+            .logfile_path
+            .file_name()
+            .unwrap()
+            .to_str()
+            .unwrap()
+            .to_string();
+        let file_appender = RollingFileAppender::builder()
+            .max_log_files(args.max_logfiles_num)
+            .rotation(Rotation::DAILY)
```

Comment:
Do we need to rotate by day? I don't think so. It's just the size rotation we care about. With no usage at all we don't need to rotate every 24 hours. Just put a specific size for rotation. Same as the one in python

Also adjust the docstring when you do that